entries/p/payfit.com.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Payfit": {
+    "domain": "payfit.com",
+    "url": "https://www.payfit.com",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```